### DISCUSSION TITLE
Support for AVIF image format.

### DISCUSSION THREAD

**Discussion by pablorosatti** [sumatrapdfreader/sumatrapdf#2473] [Ideas]
For ComicBook files (CBZ, CBR, CB7, etc) currently Sumatra supports the old image formats in addition to WEBP (30% smaller than jpeg) which was added recently, it would be great if it supports the new AVIF format (50% smaller than jpeg). jpeg), it would be the first application to do so. I often download CBR/CBZ files (containing jpg-png-gif-bmp) and convert them to CB7 (containing webp) and have managed to save a lot of space on my storage. With AVIF it would be much better.

https://www.finally.agency/blog/what-is-avif-image-format

**Comment by GitHubRulesOK** [comment]
several reasons why not one is that its not a common format across multiple containers such as html (webp) which does not work in pdf thus raises errors when try to save as pdf.
The core engine of SumatraPDF is MuPDF thus when MS or Artifex add that to their native lists it can be generically used, currently Heic/f is supported via Windows extensions, Legacy TGA AFAIK has not been dropped yet, and eps/PS are supported by Artifex GhostScript thus there are enough oddities without adding code for another, however the official answer is in https://github.com/sumatrapdfreader/sumatrapdf/issues/1520
